Library/Homebrew/cmd/test-bot.rb

```diff
@@ -221,8 +221,8 @@ def initialize(argument, options={})
       @added_formulae = []
       @modified_formula = []
       @steps = []
-      @tap = options.fetch(:tap, CoreTap.instance)
-      @repository = @tap.path
+      @tap = options[:tap]
```

Comment:
@xu-cheng What's the tap set to for Homebrew/brew PRs? It seems wrong for it to be set in that case.